src/dropdown.js

```diff
@@ -33,6 +37,14 @@ export default class extends Controller {
       !this.menuTarget.classList.contains(this.toggleClass)
     ) {
       this.menuTarget.classList.add(this.toggleClass)
+      if(this.activeClass != '') {
```

Comment:
same issue here. Any suggesstion to avoid the conditional would be great as it would make it easier to maintain for the next person.

Reply:
Will this give an error if you just always run it, such as `classList.remove(null)`

Reply:
If it works fine with `null`, you can probably just remove the conditional and always run it.

Reply:
Also, we should test this with multiple classes (which would be space separated in a string) to make sure it can handle that like `data-dropdown-active-class="bg-teal-800 text-white"`

Reply:
Thanks Chris.

I'll have another look at it. Leave it with me.

Reply:
Sounds good!

Reply:
Updates taken onboard Chris.